### DISCUSSION TITLE
Lesson 2.3: The Privacy Dilemma | Question About Quiz Answer

### DISCUSSION THREAD

**Discussion by Kiaka007** [OpenMined/courses#177] [Course: Our Privacy Opportunity] [answered]
I was thrown off a tad by quiz question number 2:
**Question:** "What is contextual integrity?"
**Answer:** "a description of when people in society feel their privacy is violated"

Reason being is that the answer only addresses when something has been violated. I guess I can't tell if this is feedback or a discussion but what I got from Helen is that "Contextual Integrity" is the expectations or line that people or a certain group draws by which we define whether privacy has been violated or been preserved based on context, so not the act of violating itself or the after effects of it being violated. I guess I need guidance on whether I interpreted that correctly and if not how I should think about it. Also content has been bomb! Great stuff you guys 🙌

**Comment by adtygan** [comment]
Yes, I feel you are right in the fact that the answer option is not comprehensive. Contextual integrity also explains the cases where one's privacy has been preserved. But I feel one cannot really describe the cases where privacy has been preserved - for one, because it's too broad, and two, it's not really necessary.
Just my take on this.

**Comment by em-blue** [comment, marked as answer]
@Kiaka007 @AdityaG09 yeah I agree, to be honest, it's a large topic and not really suited for a multiple choice question - I've decided to just remove that question entirely because it was bugging me, too

**Reply by Kiaka007** [reply]
@em-blue Gotcha ^_^
Also good point @AdityaG09 it would make sense that it isn't really used when things are going well but more when there is something that needs to be adjusted.
